includes/class-wc-emails.php

```diff
@@ -611,6 +611,19 @@ public function low_stock( $product ) {
 			return;
 		}
 
+		/**
+		 * Action hook fired to determine if the current product should trigger a
```

Comment:
@JohnTendik This is a filter vs a hook. You could remove the preface up to the `Determine` as it will be documented as a filter.